python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -585,16 +724,17 @@ async def do_disableCorrection(self, id_data):
         await asyncio.sleep(0.)  # give control back to event loop
         self.publish_enable_corrections()
 
-    async def do_setFocus(self, id_data):
-        """Set focus position.
+    async def do_setWavelength(self, id_data):
+        """Set wavelength to optimize focus.
 
         Parameters
         ----------
         id_data : `CommandIdData`
             Command ID and data
         """
-        self.assert_enabled('setFocus')
+        self.assert_enabled('setWavelength')
 
+    # FIXME: This does not appear to be used. Can it be removed?
```

Comment:
The health monitor? I had plans for this method but have not implemented them yet. I think we could keep them. Probably need a `TODO` instead of `FIXME`

Reply:
ok, is there a ticket for the work I can reference? Then i can learn about your secret plans :)

Reply:
I should probably create that ticket

Reply:
wait.. if anyone calls this then it goes into an infinite loop...
i'll comment it out and put a `pass`